<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommAudio</string>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="musicListLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <column>
         <property name="text">
          <string>Connected Servers</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>2</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="songStatusLayout">
      <item>
       <widget class="QLabel" name="left">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="right">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="rewindBtn">
        <property name="text">
         <string>Rewind</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_fast_rewind_black_24dp_2x.png</normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_fast_rewind_black_24dp_2x.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playPauseBtn">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_play_arrow_black_24dp_2x.png</normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_play_arrow_black_24dp_2x.png</iconset>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fastForwardBtn">
        <property name="text">
         <string>Fast Forward</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_fast_forward_black_24dp_2x.png</normaloff>../../Source/Repos/comm-audio/CommAudio/Resources/icons/ic_fast_forward_black_24dp_2x.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_file"/>
    <addaction name="actionAdd_folder"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_server"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuVoice_chat">
    <property name="title">
     <string>Voice chat</string>
    </property>
    <addaction name="actionStart_session"/>
    <addaction name="actionJoin_session"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVoice_chat"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_file">
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionAdd_folder">
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStart_session">
   <property name="text">
    <string>Start session</string>
   </property>
  </action>
  <action name="actionJoin_session">
   <property name="text">
    <string>Join session</string>
   </property>
  </action>
  <action name="actionAdd_server">
   <property name="text">
    <string>Add server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
